src/com/sindhu/webapplication/ToDoList.java

```diff
@@ -5,12 +5,12 @@
 import java.util.List;
 
 public class ToDoList {
-    public static  List<Long> toDoListIds(){
+    public static List<Long> toDoListIds() {
```

Comment:
just changed everything....i have wrote this function in order to get the lastId of the datebase...just realized that in order of getting whole list ids and getting the last id of the list is a stupid thing...so now wrote directly the sql query to get the last id of the table